Week4/BlockbusterRecycler/app/src/main/java/com/e/blockbusterrecycler/MovieListViewHolder.kt

```diff
@@ -0,0 +1,18 @@
+package com.e.blockbusterrecycler
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+
+class MovieListViewHolder(view: View): RecyclerView.ViewHolder(view) {
+
+    private val movieImage = itemView.findViewById<ImageView>(R.id.imageMovie)
+    private val movieTitle = itemView.findViewById<TextView>(R.id.movieTitle)
+
+    fun bindView(image: ModelMovies){
```

Comment:
I think `movie` might have been a more intuitive name than `image` but this method works fine.